packages/common/src/types.ts

```diff
@@ -45,6 +68,14 @@ export type Round = bigint;
 /** An amount of energy */
 export type Energy = bigint;
 
+/**
+ * The number of chain restarts via a protocol update. An effected
+ * protocol update instruction might not change the protocol version
+ * specified in the previous field, but it always increments the genesis
+ * index.
+ */
+export type GenesisIndex = number;
```

Comment:
Theres a bunch of other places where we also have types with a `genesisIndex` field which just use the `number` type. Maybe it would make sense to align these to all use this alias?

Reply:
Sure, I can have a look monday